File: nathan/Problema0409/problema0409.js
const prompt = require("prompt-sync")({ sigint: true });
function soma(x, y){
    return x + y;
}

var n1 = Number(prompt("Digite o primeiro número: "));
var n2 = Number(prompt("Digite o segundo número: "));
console.log(soma(n1, n2))

File: nathan/Problema0409/problema0409.java
package Problema0409;

import java.util.Scanner;

public class problema0409 {

    static int soma(int x, int y){
        return x + y;
    }
    
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Digite o primeiro número: ");
        int n1 = Integer.valueOf(input.nextLine());

        System.out.println("Digite o segundo número: ");
        int n2 = Integer.valueOf(input.nextLine());

        System.out.println(soma(n1, n2));
    }
}
